lexicons/app/bsky/richtext/facet.json

```diff
@@ -0,0 +1,39 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.richtext.facet",
+  "defs": {
+    "main": {
+      "type": "object",
+      "required": ["index", "value"],
+      "properties": {
+        "index": {"type": "ref", "ref": "#textSlice"},
+        "value": {"type": "union", "refs": ["#mention", "#link"]}
+      }
+    },
+    "mention": {
+      "type": "object",
+      "description": "A facet value for actor mentions.",
+      "required": ["did"],
+      "properties": {
+        "did": {"type": "string", "format": "did"}
+      }
+    },
+    "link": {
+      "type": "object",
+      "description": "A facet value for links.",
+      "required": ["uri"],
+      "properties": {
+        "uri": {"type": "string"}
```

Comment:
Minor note: in #587 I think the intention was to use a uri format here, but we decided to just have the at-uri format for now, which wouldn't apply here.